sharding/interface.go

```diff
@@ -95,3 +95,11 @@ type RatingReaderSetter interface {
 	//IsInterfaceNil verifies if the interface is nil
 	IsInterfaceNil() bool
 }
+
+//NodesCoordinatorCache provides the capabilities needed to store and retrieve information needed in the NodesCoordinator
+type NodesCoordinatorCache interface {
```

Comment:
no need to redefine the cache type.

Reply:
For a better decoupling, we can redefine the interface needed by this package. It could have been simply `Cacher` so it can be reused elsewhere in the sharding package.

Reply:
yes, for decoupling purposes and I also need just the Get and Put methods here. Renamed to Cacher